shared/middleware/net_sim_api.rb

```diff
@@ -69,6 +76,40 @@ def has_json_utf8_headers(request)
     get_table(shard_id, table_name).fetch(id.to_i).to_json
   end
 
+  # Delete all wires associated with (locally or remotely) a given node_id
+  #
+  # @param [String] shard_id
+  # @param [Integer] node_id
```

Comment:
Should this method be consider @private (i.e. an internal implementation detail)